daycare/test_daycare.py

```diff
@@ -3,6 +3,7 @@
 from django.core.exceptions import ValidationError
 from django.contrib.auth.models import User
 
+
 EMAIL = "test@gmail.com"
 USERNAME = "testuser01"
```

Comment:
I'm using them for the validations tests. for example: `test_daycare_user_with_invalid_name` tests are reciveing all those constants besides the name field, so `create_daycare_user` is not necessary in that case because he has a different name field.
and so on with other fields for other tests.

Reply:
You can check the test of calculating the total_price of an order, I used pytest.mark.parametrize for running several tests with different values in an easier and more elegant way.
If you have any questions, I'm available :)

Reply:
BTW I do prefer you create an issue for it to take care of it on a later PR, as it is out of this one's scope, but still need fixing.